bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/command/IntegraStatusCommand.java

```diff
@@ -52,13 +55,15 @@ public LocalDateTime getIntegraTime() {
      * @return first status byte
      */
     public byte getStatusByte1() {
+        final SatelMessage response = getResponse();
```

Comment:
I think the code will look less ugly if you just inline the call to getResponse in places where there is only one occurrence, especially with the `final` added it gets very verbose.

Reply:
Done.